kornia/models/segmentor/segmentation_models.py

```diff
@@ -47,27 +51,30 @@ def __init__(
         )
 
     def preprocessing(self, input: Tensor) -> Tensor:
-        if self.preproc_params["input_space"] == "RGB":
+        # Ensure the color space transformation is ONNX-friendly
+        input_space = self.preproc_params["input_space"]
```

Comment:
To be consistant with the image api and protos let’s call this `input_color_space`